packages/web-client/app/components/card-pay/transaction-amount/index.ts

```diff
@@ -48,6 +78,10 @@ class CardPayDepositWorkflowTransactionAmountComponent extends Component<Workflo
       balance = this.layer1Network.daiBalance;
     } else if (this.currentTokenSymbol === 'CARD') {
       balance = this.layer1Network.cardBalance;
+    } else if (this.currentTokenSymbol === 'DAI.CPXD') {
+      balance = this.layer2Network.defaultTokenBalance;
+    } else if (this.currentTokenSymbol === 'CARD.CPXD') {
+      balance = this.layer2Network.cardBalance;
```

Comment:
This extends what was already present, but maybe it could be encapsulated in the service somehow? And is `defaultTokenBalance` even correct?

Reply:
`defaultTokenBalance` in layer2 refers to the DAI.CPXD balance